src/app/admin/api-key/api-key-edit/api-key-edit.component.ts

```diff
@@ -79,8 +83,17 @@ export class ApiKeyEditComponent implements OnInit {
       );
   }
 
+  private getApiKey(id: number) {
+    this.apiKeyService.get(id).subscribe((key) => {
+      this.apiKeyRequest = new ApiKeyRequest();
```

Comment:
In line 20, apiKeyRequest is already created with new ApiKeyRequest(). Is there a reason that this is done again in this place? Furthermore, apiKeyRequest in line 20 could be declared at public compared to the other properties.